meta/main.yml

```diff
@@ -1,10 +1,11 @@
 ---
 galaxy_info:
-  author: First Last
-  description: Skeleton Ansible role
+  author: Alexander King
+  description: Ansible role that downloads and installs the Metasploit Framework.
   company: CISA Cyber Assessments
   galaxy_tags:
-    - skeleton
+    - pentest
+    - security
```

Comment:
I would add a `metasploit` tag as well.

Reply:
I amended [087d664](https://github.com/cisagov/ansible-role-metasploit/pull/5/commits/087d664f8a70cbe560f094b98ee47217b3118122) to include the tag.

Reply:
Fixed via commit 9776036ee0fbbbc4913b24d0c129ebb82a2d112b.